lib/pyszn/injection.py

```diff
@@ -150,14 +154,46 @@ def parse_attribute_injection(
                     result[filename]['environment'][attribute] = value
             except KeyError:
                 pass
+
+            # Get the topology for the filename and if it doesn't exist in the cache
+            # parse it and save it to the parsed_topologies_cache
+            if filename.endswith('.py'):
+                topology = find_topology_in_python(filename, szn_dir=szn_dir)
+                if topology is None:
+                    log.warning(
+                        ('Skipping node expansion for attribute injection in filename '
+                        '{} in the lookup path as it does not contain a TOPOLOGY or TOPOLOGY_ID '
+                        'definition.').format(filename))
+                    return []
```

Comment:
this return is not ok here, it will prevent other files to be processed, I think you meant 
```suggestion
                    break
```

Reply:
Ah yes good catch 😅